jsonrpc/server.go

```diff
@@ -56,6 +56,7 @@ func Err(code int, data any) *Error {
 		return &Error{Code: MethodNotFound, Message: "Method Not Found", Data: data}
 	case InvalidParams:
 		return &Error{Code: InvalidParams, Message: "Invalid Params", Data: data}
+
```

Comment:
Spurious diff.

Reply:
Can we remove this change since it is unrelated?

Reply:
done